compiler-core/src/type_/printer.rs

```diff
@@ -200,35 +286,70 @@ pub enum NamedTypeNames<'a> {
     Qualified(&'a str, &'a str),
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum PrintMode {
+    /// Prints the context-specific representation of a type
+    Normal,
+    /// Prints the raw type, always qualified. Useful for providing
+    /// more detail to the user.
+    Raw,
```

Comment:
I'm not sure this documentation described how this is actually used, and the name isn't super clear either. I think it's whether to show aliases?

Normal and ExpandAliases ?